SettingsList.py

```diff
@@ -3571,16 +3571,27 @@ def __init__(self, name, gui_text, min, max, default, step=1,
                          ''',
         shared         = True,
     ),
-    Checkbutton(
-        name           = 'correct_chest_sizes',
-        gui_text       = 'Chest Size Matches Contents',
-        gui_tooltip    = '''\
-            Chests will be large if they contain a major
-            item and small if they don't. Boss keys will
-            be in gold chests. This allows skipping
-            chests if they are small. However, skipping
-            small chests will mean having low health,
-            ammo, and rupees, so doing so is a risk.
+    Combobox(
+        name           = 'correct_chest_appearances',
+        gui_text       = 'Chest Appearance Matches Contents',
+        default        = 'off',
+        choices        = {
+            'off': 'Off',
+            'textures': 'Textures',
+            'sizes': 'Size and Textures'
```

Comment:
These should be all singular or all plural. 'Texture' vs 'Size and Texture' or 'Textures' vs 'Sizes and Textures'. Personally I think singular is better (matches 'Chest Appearance')

Reply:
I made it all singular for the GUI. The setting internally was all plural, and I kept it as-is there to avoid missing a spot.